pkg/apis/types/update_serving.go

```diff
@@ -19,6 +19,7 @@ type CommonUpdateServingArgs struct {
 	Tolerations   []TolerationArgs  `yaml:"tolerations"`   // --toleration
 	Shell         string            `yaml:"shell"`         // --shell
 	Command       string            `yaml:"command"`       // --command
+	ModelDirs     map[string]string `yaml:"modelDirs"`     // --data
```

Comment:
Should other types (such as custom, triton, tensorflow) support update --data since it is a common argument?